src/Symfony/Component/Messenger/Exception/MessageDecodingFailedException.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Exception;
+
+/**
+ * Thrown when a message cannot be decoded in a serializer.
+ *
+ * @experimental in 4.3
+ */
+class MessageDecodingFailedException extends \Exception implements ExceptionInterface
```

Comment:
We rarely extend `\Exception` directly. Should it be `InvalidArgumentException` instead?